CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ include(cmake/CompilerOptions.cmake)
 include(cmake/options.cmake)
 include(cmake/thirdParties.cmake)
 include(cmake/install.cmake)  # install setup (do it before adding targets)
+include(cmake/Uninstall.cmake)  # install setup (do it before adding targets)
```

Comment:
do we need the uninstall? if so, we need I would like to double check it.

Reply:
go for it. You should see that calling

make uninstall

removes all installed filed.